optionals/compat_sog/CfgVehicles/explosives.hpp

```diff
@@ -140,3 +140,91 @@ class EXPLOSIVES_PLACE(punji_03): EGVAR(explosives,Place) {
     displayName = "$STR_VN_MINE_PUNJI_01_MAG_DN";
     model = "\vn\weapons_f_vietnam\mines\punji\vn_mine_punji_03_mag";
 };
+
+class EXPLOSIVES_PLACE(punji_04): EGVAR(explosives,Place) {
+    displayName = "$STR_VN_MINE_PUNJI_04_MAG_DN";
+    model = "vn\weapons_f_vietnam_03\mines\punji_02\vn_mine_punji_04_mag";
+};
+
+class EXPLOSIVES_PLACE(punji_05): EGVAR(explosives,Place) {
+    displayName = "$STR_VN_MINE_PUNJI_05_MAG_DN";
+    model = "vn\weapons_f_vietnam_03\mines\punji_02\vn_mine_punji_05_mag";
+};
+
+// Bike mine
+class EXPLOSIVES_PLACE(bike): EGVAR(explosives,Place) {
+    displayName = $STR_VN_MINE_BIKE_MAG_DN;
```

Comment:
This and so many others have an inconsistency with the above ones: the dollar ref is unquoted here, double-quoted above.

Reply:
True, I've been copy pasting directly from the source and we don't quote stringtables by convention in here 😅